client/my-sites/stats/activity-log-day/index.jsx

```diff
@@ -33,7 +35,22 @@ class ActivityLogDay extends Component {
 			tsEndOfSiteDay,
 			requestRestore,
 		} = this.props;
-		requestRestore( tsEndOfSiteDay );
+		requestRestore( tsEndOfSiteDay, 'day' );
+	};
+
+	trackOpenDay = () => {
+		const {
+			logs,
+			moment,
+			recordTracksEvent,
+			tsEndOfSiteDay,
+		} = this.props;
+
+		recordTracksEvent( 'calypso_activity_log_day_expand', {
+			logCount: logs.length,
+			tsEndOfSiteDay,
+			utcDate: moment.utc( tsEndOfSiteDay ).format( 'YYYY-MM-DD' ),
```

Comment:
I'm not sure. Let me explain my thinking and you can check me.

UTC makes sense for me because we'll be viewing this from an analytics perspective, everything should be on the same TZ.

UTC makes sense because the analytics I forsee here are things like "What dates are people most interested in" and "As we go get farther from the current date, how do things drop off". I think both of these cases would mean comparing the UTC timestamp of the analytics event with a UTC date.

As far as I can see, site-time isn't relevant for analytics. I don't think analytics is interested (or can really aggregate things) in local site time.

Do those seem like reasonable ideas?